lib/autocapture/attribution_options.dart

```diff
@@ -0,0 +1,39 @@
+class AttributionOptions {
+  /// Web specific
+  ///
+  /// List of referrers to exclude from attribution tracking.
+  ///
+  /// Sets rules to decide which referrers to exclude from tracking as traffic source.
+  /// See [docs](https://amplitude.com/docs/sdks/analytics/browser/browser-sdk-2#track-marketing-attribution) for more information.
+  final List<String>? excludeReferrers;
+  /// Web specific
+  ///
+  /// Sets the value to represent undefined/no initial campaign parameter for first-touch attribution. The default value is "EMPTY".
+  final String? initialEmptyValue;
+  /// Web specific
+  ///
+  /// Configures Amplitude to start a new session if any campaign parameter changes. The default value is false.
+  final bool resetSessionOnNewCampaign;
```

Comment:
Also optional?  https://github.com/amplitude/Amplitude-TypeScript/blob/c415f792a98253ac60885eb1dc7e53b78ca47dcb/packages/analytics-types/src/config/browser.ts#L174-L190
```suggestion
  final bool? resetSessionOnNewCampaign;
```